### PR TITLE
Add tags and processors on GCP Compute, Firestore, PostgreSQL

### PR DESCRIPTION
**Pull request by Lucas-Feat**
### Proposed commit message
Add custom processors and tags to gcp Compute, Firestore and PostgreSQL integrations

### Checklist

- [x]  I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ]  I have verified that all data streams collect metrics or logs.
- [x]  I have added an entry to my package's changelog.yml file.
- [x]  I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

**Comment by Lucas-Feat** [comment]
Do we have to reopen a new PR after we sign the Contributor Agreement ?

Thkx

**Comment by aliabbas-elastic** [comment]
@Lucas-Feat Can you also update the same at rest places as per above suggestions :)

**Comment by Lucas-Feat** [comment]
> @Lucas-Feat Can you also update the same at rest places as per above suggestions :) Also please sign the contributor agreement else the CI won't be green

I signed it twice, i don't know what i am doing wrong.

**Comment by RJiraya** [comment]
Hello, what's missing to merge this ? Thanks !

**Review by aliabbas-elastic — APPROVED** [review]
LGTM

**Comment by RJiraya** [comment]
Hello, what's the next step to merge, and when it can be available ? Thank's !

File: packages/gcp/manifest.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: gcp
 title: Google Cloud Platform
-version: "2.33.2"
+version: "2.33.3"
```
Comment (ishleenk17) [inline packages/gcp/manifest.yml:3]: Since this is an enhancement, let's change the version to 2.34.0

**Review by ishleenk17 — COMMENTED** [review]
Changes look good. Just one nit.

**Comment by RJiraya** [comment]
Hello, is it good now ?

**Comment by RJiraya** [comment]
any update ?

**Review by ishleenk17 — APPROVED** [review]
Looks good!

**Review by ShourieG — APPROVED** [review]
LGTM

**Review by MichaelKatsoulis — APPROVED** [review]

**Pull request merged**